trieste/acquisition/multi_objective/pareto.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2020 The Trieste Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" This module contains functions and classes for Pareto based multi-objective optimization. """
+from __future__ import annotations
+
+import tensorflow as tf
+
+from ...types import TensorType
+from .dominance import non_dominated
+from .partition import prepare_default_non_dominated_partition_bounds
+
+
+class Pareto:
+    """
+    A :class:`Pareto` Construct a Pareto set.
+    Stores a Pareto set and calculates the cell bounds covering the non-dominated region.
+    The latter is needed for certain multiobjective acquisition functions.
+
+    For hypervolume-based multiobjective optimisation with n>2 objectives, this class
+    use branch and bound procedure algorithm. a divide and conquer method introduced
```

Comment:
uses a branch and bound procedure --- the divide and conquer method introduced ...

Reply:
Say what it does for n=2